contractsV5/KyberNetwork.sol

```diff
@@ -197,13 +198,15 @@ contract KyberNetwork is Withdrawable, Utils, IKyberNetwork, ReentrancyGuard {
         return true;
     }
 
-    event FeeHandlerContractSet(IFeeHandler newContract, IFeeHandler currentContract);
+    event ContractsUpdated(IFeeHandler newContract, IFeeHandler currentContract, IKyberDAO oldDAO, IKyberDAO newDAO);
```

Comment:
Keep the order consistent?
So either old contract first, then new, or new first then old, ie.
1) `IFeeHandler oldFee, IFeeHandler newFee, IKyberDAO oldDAO, IKyberDAO newDAO`
2) `IFeeHandler newFee, IFeeHandler oldFee, IKyberDAO newDAO, IKyberDAO oldDAO`

Reply:
OK
changed the logic a bit.
anyway its fixed.